canonical-web-and-design/react-components
# Remove custom styles from Spinner component

## Done

- Removes custom styles from Spinner component
- Removes unnecessary (?) inline property

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Check the Spinner storybook if it renders correctly and has all needed options
  - https://react-components-363.demos.haus/?path=/docs/spinner--default-story
- Ideally, `yarn link` this feature branch to external project that uses Spinner and check if there are no regressions


## Fixes

Fixes: https://github.com/canonical-web-and-design/vanilla-framework/issues/3145
